boot/src/test/java/com.taobao.arthas.boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package com.taobao.arthas.boot;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testreadMavenReleaseVersion() {
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(""));
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(null));
+    }
+
+    @Test
+    public void testreadAllMavenVersion() {
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(""));
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(null));
+    }
+
+    @Test
+    public void testgetRepoUrl() {
```

Comment:
Do the first two tests in this method cover any lines which are not covered by the other three?

Reply:
Removed those asserts